### DISCUSSION TITLE
Introduction

### DISCUSSION THREAD

**Discussion by Mumin-ahmed** [community/community#30392] [Other Feature Feedback, Questions, & Ideas] [answered]
Hi guys, my name is Mumin Ahmed, I'm new to the community 😉 and I have like 1001 questions to ask but I'm sure I'll get used to it.
One question tho, my github app is redirecting me to my browser anytime I click the sign in button, does anyone know the reason and how to fix it?😥

**Comment by NikhilMath** [comment, marked as answer]
It just has to do that once. Once you do that once, it should not happen again.

**Reply by Mumin-ahmed** [reply]
Can you break it down a little bit more🥲... I'm new to GitHub😭
And my first experience is my GitHub app not working.

**Reply by Mumin-ahmed** [reply]
And please how can I fix it😭

**Reply by NikhilMath** [reply]
what happens after you login in on the broswer? does the app update?

**Reply by Mumin-ahmed** [reply]
No, it just takes me there, and that's where I get to use my GitHub😭...currently I'm messaging you from opera mini

**Reply by Mumin-ahmed** [reply]
You know, it's like going to Google and you search for GitHub login and you log in, and you use GitHub there instead of using the GitHub app🥲
